### ISSUE TITLE
When a PICS is aborted the response returns a `GENERIC_ERROR` result code instead of `ABORTED`

### ISSUE THREAD

**Issue by NicoleYarroch** [smartdevicelink/generic_hmi#212]
When a PICS is aborted, either by a timeout or by closing the PICS menu without selecting anything, the response returns a `GENERIC_ERROR` result code instead of `ABORTED`.

I get: 
```
2019-10-15 14:19:28.045194 SDL Example Swift[24000:1031725] [Proxy] Message received: PerformInteraction (response), id: 111
{
    info = "UI.PerformInteraction Failed";
    resultCode = "GENERIC_ERROR";
    success = 0;
}
2019-10-15 14:19:28.046373 SDL Example Swift[24000:1031726] [Screen/ChoiceSet] Presenting choice set failed with response: PerformInteraction (response), id: 111
{
    info = "UI.PerformInteraction Failed";
    resultCode = "GENERIC_ERROR";
    success = 0;
}, 
```

When I expect something like:
```
2019-10-15 14:24:13.128034-0400 SDL Example Swift[741:125625] [Proxy] Message received: PerformInteraction (response), id: 34
{
    info = "Aborted response";
    resultCode = ABORTED;
    success = 0;
}
```

**Comment by iCollin** [comment]
The [HMI Integration Guidelines](https://smartdevicelink.com/en/guides/hmi/ui/performinteraction/) suggest you should get code: 10 (TIMED_OUT) if Core does not hear from the HMI within the timeout.

**Comment by iCollin** [comment]
Closed via #215
